dissect/target/plugins/browsers/browser.py

```diff
@@ -20,6 +20,17 @@
     ("uri", "from_url"),
     ("path", "source"),
 ]
+GENERIC_DOWNLOAD_RECORD_FIELDS = [
```

Comment:
Maybe also add a `BrowserDownloadRecord` like the `BrowserHistoryRecord` below, and add a `downloads` function to the `BrowserPlugin`.